static/gui/static/css/codemirror.css

```diff
@@ -0,0 +1,350 @@
+/* BASICS */
```

Comment:
Did you get this css file from somewhere?  If so, add a link to the original website in a comment somewhere so that we can cite the source in our documention.